server/src/main/java/org/apache/iotdb/db/service/TSServiceImpl.java

```diff
@@ -1649,15 +1632,6 @@ protected TSStatus executeNonQueryPlan(PhysicalPlan plan) {
         : RpcUtils.getStatus(TSStatusCode.EXECUTE_STATEMENT_ERROR);
   }
 
-  private TSStatus checkPathValidity(String path) {
-    if (!PATH_PATTERN.matcher(path).matches()) {
```

Comment:
Hi, could the PATH_PATTERN also be removed?

Reply:
Good catch, thanks. Indeed I just pushed an Update and if tests are okay I will merge!